packages/web/src/components/edit/fields/visibility/ReleaseDateField.tsx

```diff
@@ -29,34 +29,37 @@ export const ReleaseDateField = () => {
           label={messages.dateLabel}
           futureDatesOnly
         />
-        <HarmonyTextField
-          name='releaseDateTime'
-          label={messages.timeLabel}
-          transformValueOnBlur={(value) => {
-            if (value.includes(':')) {
-              return value
-            }
-            // add :00 if it's missing
-            const number = parseInt(value, 10)
-            if (!isNaN(number) && number >= 1 && number <= 12) {
-              return `${number}:00`
-            }
-            return value
-          }}
-        />
-        <DropdownField
-          placeholder={messages.meridianPlaceholder}
-          menu={{ items: ['AM', 'PM'] }}
-          size='large'
-          name='releaseDateMeridian'
-          dropdownInputStyle={css({
-            height: '64px !important',
-            '.ant-select-selection-item': {
-              display: 'flex',
-              alignItems: 'center'
-            }
-          })}
-        />
+        <Flex gap='l'>
+          <Box flex={1}>
```

Comment:
what is this.. why is the `Flex` not enough?

Reply:
i think because the text inputs are "input" elements which are not flex elements?

Reply:
oh no reason, just when it's a single item that doesnt need special flex stuff i do this.